task/python-sdk-azure/0.1/python-sdk-azure.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: python-azure-sdk
```

Comment:
I am trying to think, can we make it a common task which can work with different cloud providers? Having this just for azure means that we are going to have n more tasks for aws, gcp, etc

Reply:
according to me whole purpose of task is to make it small and  concise for a particular step ..  hence it will be very cumbersome to make one task for all .. better to keep it isolated

Reply:
done